backend/src/services/vacc.service.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import config from '../config';
+
+export default async function vaccAuth_URL(cid: string): Promise<boolean> {
```

Comment:
This must not be the default export. Please have a look at basically every other service we have.